07_perfect_forwarding/07_01_not_rvalue_reference/07_01_01_not_rvalue_reference.cpp

```diff
@@ -0,0 +1,20 @@
+// Describe that && not always uses in rvalue reference
+
+#include <vector>
+
+struct Some {};
+
+void f1(Some&& rRef) {}
+
+template <typename T>
+void f2(T&& tRef) {}
```

Comment:
```suggestion
void f1(Some&&) {}

template <typename T>
void f2(T&&) {}
```

Do we want to see "unused parameter" warnings?

Reply:
I don't know actually. In my point of view it looks simpler with names.

Reply:
I believe we talk not only about template parameters but also about function parameters. So, let's keep it.